### PR TITLE
Rename Node : Keep only renamed parameter

### PR DESCRIPTION
**Pull request by ghost**
Added the boolean to only keep renamed fields.
Changed are based on the SET node.
Please review and add to the change.

### CODE REVIEW

File: packages/nodes-base/nodes/RenameKeys/RenameKeys.node.ts [hunk 1]
```diff
@@ -78,23 +86,30 @@ export class RenameKeys implements INodeType {
...
+			
+			if (keepOnlyRenamed !== true) {
+				if (item.binary !== undefined) {
+					// Reference binary data if any exists. We can reference it
+					// as this nodes does not change it
+					newItem.binary = item.binary;
+				}
+				
+				// Copy the whole JSON data as data on any level can be renamed
+				newItem = {
```
Comment (pemontto) [inline packages/nodes-base/nodes/RenameKeys/RenameKeys.node.ts:109]: You're re-assigning a constant here. L97 should use the `let` keyword, or the original L81 should be brought back and both L97 and L109 should just be bare assignments.

Comment (bramknuever) [reply packages/nodes-base/nodes/RenameKeys/RenameKeys.node.ts:110]: Thanks, I messed up there. 
Fixed and tested it.

**Comment by carlin-q-scott** [comment]
@pemontto Do you have any more review feedback for this PR, or can it be merged? I need this feature for the workflow I'm trying to set up.

**Comment by michael-radency** [comment]
Hello @bramknuever I made some fixes to binary data, as it was missing from node output and they shouldn't be touched as this node does not change it

**Comment by Joffcom** [comment]
Hey @bramknuever,

Due to the age of this one and upcoming internal plans we are likely to include this functionality in a future overhaul of the Set node so I am going to close this one for now.

Thanks for the time you spent working on this though and hopefully this won't stop you from created other PRs in the future.

**Pull request closed without merging**